### PR TITLE
Fixing typos for all versions

### PR DESCRIPTION
**Pull request by pkosko**
This PR applies following changes:
- adds description for 2.3.2 and 2.4 same as in https://github.com/Samsung/tizen-docs/pull/531
- fixes typos in the documentation of all versions

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/538

**Comment by ryanhc** [comment]
Fix for the wearable web widget is here: https://github.com/Samsung/tizen-docs/pull/543

File: docs/application/web/api/5.0/device_api/wearable/tizen/bluetooth.html [hunk 1]
```diff
@@ -5864,7 +5864,7 @@ <h4>Methods</h4>
 {
```
[the remark this answers is not in the document]
Comment (pkosko) [reply docs/application/web/api/5.0/device_api/wearable/tizen/bluetooth.html]: Unfortunately there is no possibility to change this typo - it is the part of API definition - it need an ACR for change.

File: docs/application/web/api/2.3.2/device_api/wearable/tizen/systeminfo.html [hunk 2]
```diff
@@ -1029,8 +1029,7 @@ <h4>Methods</h4>
             <p>
```
[the remark this answers is not in the document]
Comment (pkosko) [reply docs/application/web/api/2.3.2/device_api/wearable/tizen/systeminfo.html]: Unfortunately there is no possibility to change this typo - both versions are the part of API definition. Until the deprecated API removal policy will be applied, we need to keep all API unchanged.

File: docs/application/web/api/2.4/device_api/mobile/tizen/systeminfo.html [hunk 3]
```diff
@@ -1085,8 +1085,7 @@ <h4>Methods</h4>
             <p>
 If a valid listenerId argument is passed that corresponds to an existing subscription,
```
[the remark this answers is not in the document]
Comment (pkosko) [reply docs/application/web/api/2.4/device_api/mobile/tizen/systeminfo.html]: Unfortunately there is no possibility to change this typo - both versions are the part of API definition. Until the deprecated API removal policy will be applied, we need to keep all API unchanged.

**Comment by mijin85cho** [comment]
Hello, @pkosko.
Thank for the answers. I have a question.

Is there any issue to do ACR for "WEARABLE_WRITST_WATCH" and "PERIPHERAL_DEGITIZER_TABLET"? The misspell issues are also sent to Native API maintainer, so they are now processing the API update steps, too.

**Comment by pkosko** [comment]
@mijin85cho there is no issue with creating new ACR - I plan to do this. It is just not doable as a "quick fix of typo" - whole ACR process need to be done for this. It need to be done separately from this PR

**Pull request merged**